core/src/main/java/greencity/mapping/TagDtoMapper.java

```diff
@@ -0,0 +1,20 @@
+package greencity.mapping;
+
+import greencity.dto.tag.TagDto;
+import greencity.entity.Tag;
+import java.util.ArrayList;
+import java.util.List;
+import org.modelmapper.AbstractConverter;
+import org.springframework.stereotype.Component;
+
+@Component
+public class TagDtoMapper extends AbstractConverter<List<Tag>, List<TagDto>> {
```

Comment:
Converter should convert Tag to TagDto and forEach must live in some other place